### PR TITLE
Separeting config from logic

### PR DESCRIPTION
**Pull request by ahangarha**
Now, config (base_url and results_per_page) are saved in `.env` file (should be copied from `.env.example`)

This could resolve the first TODO in #15

### CODE REVIEW

File: gsearch.py [hunk 1]
```diff
@@ -16,13 +18,22 @@
...
 reload(sys)
 sys.setdefaultencoding('utf-8')
 
-base_url = 'https://www.google.com.hk/'
-results_per_page = 10
+# Load config from .env file
+# TODO: Error handling
+try:
+    dotenv_path = join(dirname(__file__), '.env')
+    load_dotenv(dotenv_path)
```
Comment (meibenjin) [inline gsearch.py]: I think it's better touse find_dotenv(), this function will try to find a .env file by (a) guessing where to start using __file__ or the working directory

Comment (meibenjin) [inline gsearch.py]: I think it's better to use find_dotenv(), this function will try to find a .env file by (a) guessing where to start using file or the working directory - allowing this to work in non-file contexts such as IPython notebooks and the REPL, and then (b) walking up the directory tree looking for the specified file – called .env by default

**Comment by ahangarha** [comment]
That is a good idea. I will make this change

**Comment by ahangarha** [comment]
I have made the change. Have a look please

**Pull request merged**